src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Policy/UpdateAzureRmRecoveryServicesAsrPolicy.cs

```diff
@@ -345,20 +345,13 @@ private void HyperVToAzurePolicyObject()
                     Utilities.GetMemberName(() => this.RecoveryAzureStorageAccountId))
                     ? this.RecoveryAzureStorageAccountId
                     : replicationProviderSettings.ActiveStorageAccountId;
-            this.encryption =
-                this.MyInvocation.BoundParameters.ContainsKey(
-                    Utilities.GetMemberName(() => this.Encryption)) ? this.Encryption
-                    : (string.Compare(
-                           replicationProviderSettings.Encryption,
-                           "Disabled",
-                           StringComparison.OrdinalIgnoreCase) ==
-                       0 ? Constants.Disable : Constants.Enable);
 
             var hyperVReplicaAzurePolicyInput = new HyperVReplicaAzurePolicyInput
             {
                 ApplicationConsistentSnapshotFrequencyInHours =
                     this.applicationConsistentSnapshotFrequencyInHours,
-                Encryption = this.encryption,
+// encryption is removed from latest API . Service is setting the default vault internally.
```

Comment:
Was `Encryption` exposed to the PowerShell user? Is this a breaking change?

Reply:
This field should have been already deprecated in older PowerShell and older API versions. (Service stopped honouring this field for several months already). This PR contains changes that use new api-version for APIs. 

@viverm is this field no longer available in new api? Is this why this is removed?

Reply:
we cann't remove it from cmdlet but as service is not taking that input anymore , we are not populating it.

It is not the breaking change as service was not honoring it earlier . @MiYanni